App/components/Image/image.tsx

```diff
@@ -1,23 +1,34 @@
 import React, { ReactNode } from 'react'
+import { ImageSourcePropType } from 'react-native'
 import * as Styled from './image.styled'
 
-type Props = {
+export type Props = {
   children: ReactNode
   image: string
   width: number
   height: number
-  border?: string
-  borderRadius?: number
+  border?: string | undefined
+  borderRadius?: number | undefined
 }
 
-const Image = ({ children = '', image = '', width = 0, height = 0 }: Props) => {
+const Image = ({
+  children = '',
+  image = '',
+  width = 0,
+  height = 0,
```

Comment:
Consider removing default values for it and making them optional: I suppose react native will set their height and width by default as it is.